Makefile

```diff
@@ -853,6 +853,11 @@ bench: bench-net bench-http bench-fs bench-tls
 
 bench-ci: bench
 
+mdlint:
+	@echo "Running Markdown linter..."
+	$(NODE) tools/remark-cli/cli.js -q \
+	  doc/api doc/api_assets doc/guides src lib benchmark tools/doc/ tools/icu/
```

Comment:
Is there a reason why are other stuff from `doc` ignored (e.g. `onboarding.md` and changelogs)?